src/publish-registry.ts

```diff
@@ -20,20 +20,20 @@ Generated by [types-publisher](https://github.com/Microsoft/types-publisher).`;
 
 if (!module.parent) {
 	const dry = !!yargs.argv.dry;
-	done(getDefinitelyTyped(Options.defaults).then(dt => main(dt, dry, new UncachedNpmInfoClient())));
+	done(async () => {
+		const dt = await getDefinitelyTyped(Options.defaults);
+		await main(dt, await AllPackages.read(dt), dry, new UncachedNpmInfoClient());

```

Comment:
since I'm not familiar with async syntax, I want to confirm: the only substantive change here is to add a call to `AllPackages.read(dt)`, right? I think everything else is a switch from `.then(dt =>` ... to `dt = await ...`, right?